tests/lib/rules/sort-prop-types.js

```diff
@@ -325,8 +325,47 @@ ruleTester.run('sort-prop-types', rule, {
       '};',
       'First.propTypes = propTypes;'
     ].join('\n')
+  }, {
+    code: `
+      class Component extends React.Component {
+        render() {
+          return <div />;
+        }
+      }
+      Component.propTypes = {
+        x: PropTypes.any,
+        y: PropTypes.any,
+        z: PropTypes.shape({
+          c: PropTypes.any,
+          C: PropTypes.string,
+          a: PropTypes.any,
+          b: PropTypes.bool,
+        }),
+      };
+    `
+  }, {
+    code: `
+      class Component extends React.Component {
+        render() {
+          return <div />;
+        }
+      }
+      Component.propTypes = {
+        a: PropTypes.any,
+        b: PropTypes.any,
+        c: PropTypes.shape({
+          c: PropTypes.any,
+          ...otherPropTypes,
+          a: PropTypes.any,
+          b: PropTypes.bool,
+        }),
+      };
+    `,
+    options: [{
+      sortShapeProp: true
+    }]
   }],
-
+  // Invalid code, should not be validated
```

Comment:
What's the reason for this comment?

Reply:
I think I saw the comment as an intended marking of where the invalid test cases began and that it was in the wrong place. Looking back over it, the comment was actually in the right spot. I'll rebase to put this back.